generate/generate.go

```diff
@@ -39,180 +39,17 @@ func New() Generator {
 			OS:   runtime.GOOS,
 			Arch: runtime.GOARCH,
 		},
-		Root: rspec.Root{
-			Path:     "",
-			Readonly: false,
-		},
 		Process: rspec.Process{
-			Terminal: false,
-			User:     rspec.User{},
+			User: rspec.User{
+				UID: 0,
+				GID: 0,
```

Comment:
These are the Go zero values, so you can stick with `User:     rspec.User{}`.  I'm not opposed to explicitly setting them (like you're currently doing in this PR) though.